sass/main.scss

```diff
@@ -82,6 +82,7 @@ img {
 }
 
 .slide_dosomething {
```

Comment:
Our convention on the front-end team is to use dashes in class names, and BEM-style child/modifier naming. Not necessary, but could be nice to keep things consistent (and easier for on boarding if anyone else needs to jump into the codebase).